cdk.json

```diff
@@ -64,6 +64,23 @@
       "CPU_SIZE":"512",
       "MEMORY_SIZE":"2048"
     },
+    "testing": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:v24.5.1",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "3838",
+      "TAGS": {
+        "CostCenter": "NO PROGRAM / 000000"
+      },
+      "STACK_NAME_PREFIX": "dca-testing",
+      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/a3b7c804-c1fc-4e58-bbc6-541ef2d65816",
+      "VPC_CIDR": "12.0.0.0/4",
```

Comment:
@zaro0508 can you confirm this CIDR is good?

Reply:
No, this should be a private address (RFC 1918). If you want something close to the other environments then `10.255.77.0/24` is available.

Reply:
ok, thank you for checking that.